### PR TITLE
VAGOV-6109: fix timezone dates; send correct date to timezone filter for events

### PR DESCRIPTION
**Pull request by aurora-a-k-a-lightning**
## Description
 fix timezone dates; send correct date to timezone filter for events

## Testing done
locally with prod and staging data

## Screenshots
image

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/site/filters/liquid.js [hunk 1]
```diff
@@ -19,23 +19,17 @@ module.exports = function registerFilters() {
...
-      const prettyTime = moment(diffToMoment).format(format);
-      const prettyTimeFormatted = prettyTime
-        .replace(/AM/g, 'a.m.')
-        .replace(/PM/g, 'p.m.');
-
-      return prettyTimeFormatted;
+      const timeZoneDate = new Date(dt).toLocaleString('en-US', {
+        timeZone: tz,
+      });
+      return prettyTimeFormatted(timeZoneDate, format);
```
Comment (ElijahLynn) [inline src/site/filters/liquid.js]: Should this be a new inline function since it is only used once?

Comment (ElijahLynn) [reply src/site/filters/liquid.js]: Trivial, btw.

**Review by ElijahLynn — APPROVED** [review]

File: src/site/filters/liquid.js (hunk 1 above)
Comment (ahay-agile6) [reply src/site/filters/liquid.js]: do you mean `prettyTimeFormatted`?

**Review by U-DON — APPROVED** [review]

File: src/site/filters/liquid.js (hunk 1 above)
Comment (ahay-agile6) [reply src/site/filters/liquid.js]: If so, it is used down below. That's why i abstracted it out. Was using the same methods to achive the same time formatted string in at least two places

**Pull request merged**